pandas-stubs/io/formats/style.pyi

```diff
@@ -40,6 +41,16 @@ from pandas.io.formats.style_render import (
     Subset,
 )
 
+class SeriesFunc(Protocol):
```

Comment:
I would prefer renaming it to `_SeriesFunc` or put it in `_typing` so that users don't think they can import it.